addons/web/static/src/views/fields/field.js

```diff
@@ -416,6 +418,7 @@ export class Field extends Component {
             let fieldInfo = this.props.fieldInfo;
             readonly =
                 readonly ||
+                this.offlineStatus.offline || // disable edition in offline mode
```

Comment:
If the field does not have a prop `fieldInfo` then it could be editable offline. Is that what we want?